README.md

```diff
@@ -1,41 +1,54 @@
 # Welcome to the scheduled events lab!
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
 
-## About the codebase
+This lab is an exercise to check your knowledge on scheduled events. It is automatically graded via a workflow once you have completed the instructions.
+
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
+
+## About this lab
+
+:warning: A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and **altering their contents will affect the repository's ability to assess your lab and give feedback.**
```

Comment:
We may want to explicitly call out `.github/workflows/grading.yml` here, as the other workflows inside `.github` are the ones the learner will engage with.

Reply:
Agreed!